lib/i18n/_missing_translations.yaml

```diff
@@ -137,27 +137,27 @@ de:
 es:
   home:
     tooltips:
-      exportNote(OUTDATED): Export note
+      exportNote(OUTDATED): Exportar nota
     moveNote:
-      moveNotes(OUTDATED): Move $n notes
-      multipleRenamedTo(OUTDATED): "The following notes will be renamed:"
-      numberRenamedTo(OUTDATED): $n notes will be renamed to avoid conflicts
+      moveNotes(OUTDATED): Mover $n notas
+      multipleRenamedTo(OUTDATED): "Renombrar las siguientes notas"
+      numberRenamedTo(OUTDATED): $n notas serán renombradas para evitar conflictos
   settings:
     prefLabels:
-      recentColorsLength(OUTDATED): How many recent colors to store
+      recentColorsLength(OUTDATED): Número de colores recientes a almacenar
   editor:
     pens:
-      pencil(OUTDATED): Pencil
-      shapePen(OUTDATED): Shape pen
+      pencil(OUTDATED): Bolígrafo
+      shapePen(OUTDATED): Bolígrafo con formas
```

Comment:
The Spanish translation already has `ballpointPen: Bolígrafo`. Are you able to reword pencil, like to Lápiz?